File: SPA/FrontEnd/src/app/Models/CantHuespedes/cant-huespedes.ts
export class CantHuespedes {
    cantAdultos: number;
    cantNinios: number;
    cantBebes: number;
    cantJubilados: number;

    constructor
        (CantAdultos: number, CantNinios: number,
            CantBebes: number, CantJubilados: number) {
        this.cantAdultos = CantAdultos;
        this.cantNinios = CantNinios;
        this.cantBebes = CantBebes;
        this.cantJubilados = CantJubilados;
    }
}

File: Domain/CantHuespedes.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class CantHuespedes
    {
        public int Id { get; set; }
        public int CantAdultos { get; set; }
        public int CantNinios { get; set; }
        public int CantBebes { get; set; }
        public int CantJubilados { get; set; }

        public int CapacidadTotal()
        {
            return CantAdultos + CantBebes + CantJubilados + CantNinios;
        }
    }
}
